app/controllers/api/v1x0/requests_controller.rb

```diff
@@ -28,7 +28,7 @@ def index
       private
 
       def request_params
-        params.permit(:name, :requester, :content => {})
+        params.permit(:name, :description, :requester, :content => {})
```

Comment:
@bzwei Are any changes required to `openapi.json`  for this?

Reply:
No. The bug was open because the spec says description is optional, but code ignored it.